index.html

```diff
@@ -20,79 +20,80 @@
-          <p>Call me</p>
-        </a>
-      </div>
-    </section>
+<!-- Projects section -->
+<section id="projects">
+  <p>These are some of my projects</p>
+  <div class="projects">
+    <a href="" target="_blank" class="project-tile" rel="noopener">
+      <img src="./assets/FFC-1-SurveyForm.png" alt="Freecodecamp-SurveyForm">
+      <p><span>&lt;</span><span>Survey Form</span><span>&gt;</span></p>
+    </a>
+    <a href="" target="_blank" class="project-tile" rel="noopener">
+      <img src="./assets/FFC-2-TributePage.png" alt="Freecodecamp-TributePage">
+      <p><span>&lt;</span><span>Tribute Page</span><span>&gt;</span></p>
+    </a>
+    <a href="" target="_blank" class="project-tile" rel="noopener">
+      <img src="./assets/FFC-3-TechnicalDocumentationPage.png" alt="Freecodecamp-DocumentationPage">
+      <p><span>&lt;</span><span>Documentation Page</span><span>&gt;</span></p>
+    </a>
+    <a href="" target="_blank" class="project-tile" rel="noopener">
+      <img src="./assets/FFC-4-ProductLandingPage.png" alt="Freecodecamp-LandingPage">
+      <p><span>&lt;</span><span>Landing Page</span><span>&gt;</span></p>
+    </a>
+    <a href="" target="_blank" class="project-tile" rel="noopener">
+      <img src="./assets/FFC-5-CitySkyline.png" alt="Freecodecamp-CitySkyline">
+      <p><span>&lt;</span><span>City Skyline</span><span>&gt;</span></p>
+    </a>
+    <a href="" target="_blank" class="project-tile" rel="noopener">
+      <img src="./assets/FFC-6-Magazine.png" alt="Freecodecamp-Magazine">
+      <p><span>&lt;</span><span>Magazine</span><span>&gt;</span></p>
+    </a>
+  </div>
+  <a href="" target="_blank" class="show-all" rel="noopener">Show All
+    <i class="ri-arrow-drop-right-line ri-2x"></i>
+  </a>
+</section>
+
+<!-- Contact section -->
+<section class="contact-section" id="contact">
+  <h1>Let's work together...</h1>
+  <p class="coffee-question">How do you take your coffee?</p>
+  <div class="social-medias">
+    <a href="https://www.facebook.com" target="_blank" class="social-media" rel="noopener">
+      <i class="ri-facebook-box-fill ri-2x"></i>
+      <p>Facebook</p>
+    </a>
+    <a href="https://www.github.com" target="_blank" class="social-media" id="profile-link" rel="noopener">
+      <i class="ri-github-fill ri-2x"></i>
+      <p>Github</p>
+    </a>
+    <a href="https://www.freecodecamp.com" target="_blank" class="social-media" rel="noopener">
+      <i class="ri-fire-fill ri-2x"></i>
+      <p>Free Code Camp</p>
+    </a>
+    <a href="https://www.gmail.com" target="_blank" class="social-media" rel="noopener">
+      <i class="ri-at-line-fill ri-2x"></i>
+      <p>Send a mail</p>
+    </a>
+    <a href="https://www.whatsapp.com" target="_blank" class="social-media" rel="noopener">
+      <i class="ri-cellphone-fill ri-2x"></i>
+      <p>Call me</p>
+    </a>
+  </div>
+</section>
 
-    <footer>Created by Moises & Cristian, April 2023</footer>
+    <article>Created by Cristian Villa, 2023</article>
```

Comment:
ERROR: Should be footer instead of article

Reply:
> ERROR: Should be footer instead of article
:red_circle: You have well asked for this change 👌. But I think this comment is not clear enough to the reader **Why** footer should be used instead of article 

You  could say: :arrow_down: 
- [ ] Great job so far! 👍 I kindly recommend using the <footer> tag instead of the <article> tag in this context Using the <footer> tag instead of the <article> tag is appropriate in certain situations. The <footer> tag defines the footer section of a webpage, typically containing information like copyright notices, contact details, navigational links, or other relevant information specific to the page or site. It appears at the bottom of the page. In contrast, the <article> tag is used for self-contained, standalone content that can be independently distributed, such as blog posts or news articles. By using the appropriate tag for each section, you ensure better organization, accessibility, and adherence to HTML best practices.

- Briefly for every section that needs an update try to point it out and tell the student what changes you expect from the section and why the change is needed.